solana_test.go

```diff
@@ -0,0 +1,97 @@
+package coincodec
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/wealdtech/go-slip44"
+)
+
+func TestSolanaEncodeToBytes(t *testing.T) {
+	tests := []TestcaseEncode{
+		{
+			name:   "Normal1",
+			input:  "2gVkYWexTHR5Hb2aLeQN3tnngvWzisFKXDUPrgMHpdST",
+			output: "18f9d8d877393bbbe8d697a8a2e52879cc7e84f467656d1cce6bab5a8d2637ec",
+		},
+		{
+			name:   "Normal2",
+			input:  "2bUBiBNZyD29gP1oV6de7nxowMLoDBtopMMTGgMvjG5m",
+			output: "17b02c16bf792e54b606db6c2b10a24647a3e96215f5450186e183f57caaf0d0",
+		},
+		{
+			name:   "Normal3",
+			input:  "H4JcMPicKkHcxxDjkyyrLoQj7Kcibd9t815ak4UvTr9M",
+			output: "ee93a4f66f8d16b819bb9beb9ffccdfcdc1412e87fee6a324c2a99a1e0e67148",
+		},
+		{
+			name:   "Normal4",
```

Comment:
weird, my comment was missing, test `Normal4` is same as `Normal1`

Reply:
Oops.  I will remove 4&5 in next PR.